.jenkins/Packaging.Jenkinsfile

```diff
@@ -48,11 +52,23 @@ def WindowsPackaging(String build_type) {
     }
 }
 
-parallel "1604 SGX1FLC Package Debug" :          { LinuxPackaging('1604', 'Debug') },
+try{
+    oe.emailJobStatus('STARTED')
+
+    parallel "1604 SGX1FLC Package Debug" :          { LinuxPackaging('1604', 'Debug') },
```

Comment:
Nit: fix whitespace alignment

Reply:
Fixed, per my understanding. Is this convention/standard documented somewhere?

Reply:
@chrisrod-MSFT Usually we let clang-format take care of the formatting so we don't have to worry about it; the guideline for anything else not c/c++ is to conform to the existing formatting within the document for consistency.